### ISSUE TITLE
Hourly Vibration

### ISSUE THREAD

**Issue by okuzasongs** [stefanheule/obsidian#19]
With quiet period option. Thanks.

**Comment by stefanheule** [comment]
Good idea, but I have mostly moved on from this project.  If you (or somebody else) wants to contribute the chance, I'm happy to merge it, though.  Other than that, it is unlikely that I will get around to doing it myself.
